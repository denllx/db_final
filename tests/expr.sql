CREATE DATABASE OOP;
USE OOP;
CREATE TABLE oop_info(stu_id INT NOT NULL, PRIMARY KEY(stu_id), stu_name CHAR,grade INT);
INSERT INTO oop_info(stu_id, stu_name,grade) VALUES (2018011243, "a",100);
INSERT INTO oop_info(stu_id, stu_name,grade) VALUES (2018011043, "a",90);
INSERT INTO oop_info(stu_id, stu_name,grade) VALUES (2018010243, "a",92);
INSERT INTO oop_info(stu_id, stu_name,grade) VALUES (2018001344, "b",93);
INSERT INTO oop_info(stu_id, stu_name,grade) VALUES (2018001394, "b",99);
INSERT INTO oop_info(stu_id, stu_name,grade) VALUES (2018011445, "c",98);
SELECT MAX(grade) from oop_info;
SELECT MAX(grade) from oop_info GROUP BY stu_name;
SELECT MIN(grade) from oop_info;
SELECT MIN(grade) from oop_info GROUP BY stu_name;
