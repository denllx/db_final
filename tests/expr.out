MAX(grade)
100
stu_name	max(grade)
a	100
b	99
c	98
stu_name	max(grade)
a	100
b	99
c	98
MIN(grade)
90
stu_name	min(grade)
a	90
b	93
c	98
stu_name	min(grade)
a	90
b	93
c	98
